backend/app/views/spree/admin/orders/_carton.html.erb

```diff
@@ -50,7 +50,12 @@
       <tr class="show-tracking total">
         <td colspan="5">
           <% if carton.tracking.present? %>
-            <strong><%= Spree::Carton.human_attribute_name(:tracking) %>:</strong> <%= carton.tracking %>
+            <strong><%= Spree::Carton.human_attribute_name(:tracking) %>:</strong>
+            <% if carton.tracking_url.nil? %>
```

Comment:
I'd probably just write this as
```ruby
<% if carton.tracking_url %>
  <%= link_to carton.tracking, carton.tracking_url, target: "_blank" %>
<% else %>
  <%= carton.tracking %>
<% end %>
```
but either way this seems fine.

Reply:
Updated, Thank you 😄